### ISSUE TITLE
Cloud foundry deploy failing in builds

### ISSUE THREAD

**Issue by jt-nti** [hyperledger/composer#3153]
cf push command is currently failing with a symlink error, e.g.

```
cf push "composer-playground${PLAYGROUND_SUFFIX}" -c "node cli.js" -i 2 -m 128M --no-start
Pushing app [36;1mcomposer-playground-next-unstable[0m to org [36;1m[secure][0m / space [36;1m[secure][0m as [36;1m[secure][0m...
Getting app info...
Updating app with these attributes...
  name:                composer-playground-next-unstable
  path:                /home/travis/build/hyperledger/composer/packages/composer-playground
  buildpack:           SDK for Node.js(TM) (ibm-node.js-8.9.0, buildpack-v3.16.1-20171207-1520)
  command:             node cli.js
  disk quota:          1G
  health check type:   port
  instances:           2
  memory:              128M
  stack:               cflinuxfs2
  env:
    COMPOSER_CONFIG
  routes:
    composer-playground-next-unstable.mybluemix.net

Updating app [36;1mcomposer-playground-next-unstable[0m...
Mapping routes...
Comparing local files to remote cache...
Packaging files to upload...
Uploading files...

 0 B / 2.70 MiB [------------------------------------------------------]   0.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [=================================================] 100.00%
 2.70 MiB / 2.70 MiB [==============================================] 100.00% 1s

Waiting for API to complete processing files...
Job (cee07f97-fb78-4332-8e7b-477d70126f99) failed: The app upload is invalid: Symlink(s) point outside of root folder
[31;1mFAILED[0m
Script failed with status 1
```

**Comment by jt-nti** [comment]
I _think_ this is a combination of lerna (creates symlinks in node_modules) and the latest version of the cf cli.

It seems like older versions of the cf command, e.g. 6.33.0+a345ea34d.2017-11-20, worked even when there were symlinks in the node_modules directory. Cloud foundry needs to do an npm install anyway since lerna has hoist all the node modules out of the composer-playground package. I guess the symlinks just got ignored and all was well.

Unfortunately the latest version of the cf command, 6.34.0+f53d03a5e.2018-01-09 (which gets installed by our build script), doesn't like symlinks that point outside the app folder. (Confusingly it doesn't complain if the cf push is done with a manifest, but you do get the error without a manifest!)

**Comment by jt-nti** [comment]
Fixed in master and 0.16.x
